src/AdoptedStyleSheetsArray.ts

```diff
@@ -0,0 +1,37 @@
+import type ConstructedStyleSheet from './ConstructedStyleSheet.js';
+import type Location from './Location.js';
+import type { UnknownFunction } from './shared.js';
```

Comment:
kinda. you don't need to checkout the transpiled javascript files but like these imports:

```suggestion
import type { UnknownFunction } from './shared.ts';
```
Would be cool if its 1:1 imports

Reply:
Hey, sorry, I missed your comment for some reason.

If I understand correctly, you think I'm importing transpiled JavaScript files instead of directly importing TypeScript files. However, this is not the case. The `.js` extension is just an [ESM convention](https://www.typescriptlang.org/docs/handbook/esm-node.html), so I'm actually referring to the `.ts` file even though I'm using the `.js` extension. Also, according to the specification, another extension besides `.js` is not allowed (until the [import attributes](https://github.com/tc39/ecma262/pull/3057) proposal is accepted).

Reply:
I guess I got it. So all I need is to have additional `adoptedStyleSheets.ts` in the build folder to support Deno. I was kinda confused by your examples of my code 😅 Didn't get that it was an example.

Ok, I'll add it when I have time to fix this PR.

Reply:
no in deno you don't need a build folder i would like to just import it via the github raw cdn
so i just import the stuff form src

thats what i mean with using allowImportingTsExtensions so it actually is in the src folder correct not after the build

:D hope i made it clear